source/includes/opts/migrationName.rst

```diff
@@ -0,0 +1,13 @@
+.. reference/configuration.txt
+.. reference/mongosync.txt
```

Comment:
**Question**

Are these comments intentional or just leftover by accident?

Reply:
This file was removed and being restored from [this commit](https://github.com/mongodb/docs-cluster-to-cluster-sync/commit/a230d124796c2b1c38fc72437dc681d05cb31f57), and it was previously approved + merged with them there. I'm assuming it's left there to track which pages the include is used in.